ops/terraform/env/mgmt/scripts/transfer-quicksight-permissions.sh

```diff
@@ -16,6 +16,16 @@ readonly aws_account_id
 sole_owner_arn="${SOLE_OWNER_ARN}"
 readonly sole_owner_arn
 
+check_user_exists() {
+  users_found=$(aws quicksight list-users --aws-account-id "$aws_account_id" --namespace default \
+    --query "UserList[?Arn=='$sole_owner_arn']" | jq length)
```

Comment:
It's a little early for Halloween... are you trying to [spook your reviewers with this unquoted command substitution](https://github.com/anordal/shellharden/blob/master/how_to_do_things_safely_in_bash.md#the-first-thing-to-know-about-bash-coding)?

I'm surprised that this doesn't trigger [SC2046](https://www.shellcheck.net/wiki/SC2046). 🤔

Reply:
Whoops, will fix. Thanks!